pipeline/tracking.py

```diff
@@ -118,12 +82,21 @@ class Whisker(dj.Part):
 
 # ---- Processed Whisker data ----
 
+@schema
+class WhiskerProcessingParams(dj.Lookup):
+    definition = """
+    param_id: int
+    ---
+    
+    """
+
```

Comment:
Let's talk about it in the meeting

Reply:
```suggestion
class WhiskerProcessingParams(dj.Lookup):
    definition = """
    param_id: varchar(30)
    ---
    param_description: varchar (2000)
    param_value_1: decimal(6, 2) # first parameter value
    param_value_2: decimal(6, 2) # second parameter value
    """
    # Processing is performed on Angle_raw values, that contain the native whisker angle values.
    # Unless otherwise mentioned, filtering is done using Butterworth filter with passband ripple of at least 3dB and stopband attenuation of at least 20dB, so that order N and natural frequency Wn are [N,Wn] = buttord( Wp, Ws, 3, 20);
    contents = [
        ('SetPoint_ComputeParams', 'stopband edge frequency for low-pass threshold ','4',''),
        ('Angle_BP_ComputeParams', 'passband and stopband edge frequencies to smooth out high frequencies and remove set point ','4','20'),
        ('Angle_ComputeParams', 'passband edge frequencies to smooth out high frequencies ','40',''),
        ('Phase_ComputeParams', 'passband and stopband edge frequencies for 2nd order Butterworth filter applied before Hilbert transform ','8','30')
        ]
     
    # VP: for documentation, below are the three pre-processing steps performed in Matlab to obtain Angle_raw values (these steps should stay offline, at least for now). See also WhiskerAngleSmoothFill.m
    # 1/ find and replace outliers
    #     thetas = filloutliers(thetas,'spline','movmedian',20)';
    # 2/ smooth values
    #     thetas(whiskerNum,:)=smoothdata(thetas(whiskerNum,:),'rloess',7);
    # 3/ fill missing / NaNs values (if any)
    #     fillDim=find(size(thetas(whiskerNum,:))==max(size(thetas(whiskerNum,:))));
    #     thetas(whiskerNum,:)=fillmissing(thetas(whiskerNum,:),'spline',fillDim,'EndValues','nearest');
```

Reply:
@vncntprvst , based on your comments above, I think something like this would work, and may offer better clarity:

```
@schema
class WhiskerProcessingParams(dj.Lookup):
    definition = """
    param_set: varchar(30)  # name of this paramset used for whisker processing 
    ---
    set_point: float       # (hz)  stopband edge frequency for low-pass threshold
    angle_passband: float  # (hz)  passband edge frequencies to smooth out high frequencies and remove set point 
    angle_stopband: float  # (hz)  stopband edge frequencies to smooth out high frequencies and remove set point 
    phase_passband: float  # (hz)  passband edge frequencies for 2nd order Butterworth filter applied before Hilbert transform 
    phase_stopband: float  # (hz)  stopband edge frequencies for 2nd order Butterworth filter applied before Hilbert transform 
    """
```

Reply:
@ttngu207 Yes, that's much better!

Reply:
@ttngu207 do you want to include that in the incoming PR? If so, I'll merge this one now. Else, I'll wait for your commit.

Reply:
I can include that in the next PR, along with the ingestion structure. Please merge this one for now so it's easier for you to review the next